src/main/definitions/definition.ts

```diff
@@ -31,29 +31,29 @@ export const enum HubCaseState {
 }
 
 export const enum ClaimTypeDiscrimination {
-  AGE = 'age',
-  DISABILITY = 'disability',
-  ETHNICITY = 'ethnicity',
-  GENDER_REASSIGNMENT = 'genderReassignment',
-  MARRIAGE_OR_CIVIL_PARTNERSHIP = 'marriageOrCivilPartnership',
-  PREGNANCY_OR_MATERNITY = 'pregnancyOrMaternity',
-  RACE = 'race',
-  RELIGION_OR_BELIEF = 'religionOrBelief',
-  SEX = 'Sex (Including equal pay)',
-  SEXUAL_ORIENTATION = 'sexualOrientation',
+  AGE = 'Age',
```

Comment:
We need to discuss this ClaimTypeDiscrimination , ClaimTypePay , look at my PR in code review https://github.com/hmcts/et-sya-api/pull/135/files#diff-5fd8824a7af98622d5fed3d065b200966dbd0823cc4d4f73e50cba47837199a8  
We have to match them somehow